tools/submission/submission_checker.py

```diff
@@ -776,6 +776,33 @@ def uses_early_stopping(self, scenario):
         return (
             scenario in ["Server", "SingleStream", "MultiStream"]
         )
+    
+    def requires_equal_issue(self, model, division):
+        return (
+            division in ["closed", "network"] and
```

Comment:
@nv-alicheng to review as well. I think these should ideally be global variables instead of functions for easier readability, is that right?

Reply:
Those are taken from the root folders, and are checked to be only `"closed", "network" or "open"`